.github/workflows/benchmark.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
         run: |
           julia --project=benchmark -e 'using Pkg;
           Pkg.rm("EpiAware");
+          Pkg.rm("BenchmarkCI");
+          Pkg.add(url="https://github.com/SamuelBrand1/BenchmarkCI.jl");
```

Comment:
Why not update the project toml?

Reply:
I'm following the pattern in the BenchmarkCI README.

https://github.com/tkf/BenchmarkCI.jl

Reply:
that doesn't have a rm?

Reply:
and we didnn't follwo trhat patter up to now as we hade the project toml?

Reply:
Its now failing in ways I really don't expect. Putting in the `[sources]` section means that CI failing on `resolve` with an error that suggests that it can't register PkgBenchmark, which is really strange